app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{" + f"Name: {self.name}, Health: {self.health}, " \
+                     f"Hidden: {self.hidden}" + "}"
```

Comment:
1. Use brackets instead of `\`
2. Why do you use concatenation? Just use f-string

Reply:
because if f-string we cannot use single "{" even with "\". Or I dont know only?)

Reply:
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/131271125/a82964d7-fd0d-4636-96f7-982b5b97e44a)

Reply:
if in one row then it is too long for flake8

Reply:
1. You still can use brackets instead of `\` for line separation
2. Use `{{` and `}}` to make everything in one f-string.

Reply:
You still can use brackets instead of \ for line separation. I don't get what does that mean, which brackets???)